neqo-qpack/src/encoder.rs

```diff
@@ -257,28 +271,44 @@ impl QPackEncoder {
         }
     }
 
-    fn change_capacity(&mut self, value: u64) -> Res<()> {
+    fn need_change_capacity(&mut self, value: u64) {
```

Comment:
Names are hard, but I think that a function called `change_capacity` is very clear.  If anything, the variable name is what needs changing, to `next_capacity` or something perhaps.